### ISSUE TITLE
Add compatibility with Networkx v2.0

### ISSUE THREAD

**Issue by mstubb** [Teichlab/tracer#53]
Quite a lot has changed in Networkx v2.0 (https://networkx.github.io/documentation/stable/release/migration_guide_from_1.x_to_2.0.html) and it breaks `summarise`. 

For now, the requirements file specifies v1.11 exactly but we need to add v2.0 compatibility.

**Comment by JLTrincado** [comment]
Hi,

Is this enhancement included in v0.6?

Thanks for your work and congratulations.

Best.

**Comment by mstubb** [comment]
No, unfortunately this issue remains open. Everything works fine with Networkx v1.11 though.
